list_files/tckma.list

```diff
@@ -418,7 +418,7 @@ MD US40 I-81 MD2
 MD US40 I-895 MD/DE
 MD US50Red US219 MD560
 MD US50 DC/MD MD528
-MD US219 WV/MD US40Alt
+MD US219 WV/MD I-68(22) # No idea why US40Alt is no longer a waypoint 

```

Comment:
US219 was relocated on 2021-05-21.
https://forum.travelmapping.net/index.php?topic=4303
There's now a grade separation over US40Alt with no access.

There's a note near the bottom of your .log file:
`2021-05-21 | (USA) Maryland | US 219 | md.us219 | Relocated onto new bypass roadway north of I-68 exit 22 near Grantsville`

--

Routes that have been updated since (or on the same day as) your last .list file update have notes at the top of your .log along with the errors. For example, yakra.log says:
`Route updated 2021-08-18: PE TCHPEI`
If you see these, you can scroll down to the bottom and see the most recent update for all your .listed routes. It's sorted chronologically, so new ones will be at the very bottom.

ISTR @michihdeu created a forum topic for this but I'm having no luck finding it.

Reply:
@yakra I didn't. I only changed the manual:
https://travelmapping.net/participate.php#errorlog
https://github.com/TravelMapping/Web/pull/641

If memory serves, I asked your anywhere on Github for notifying the change on the forum but you didn't. I think because of any minor open issue. I waited for months.... maybe I was pissed(?).... well, and thought that there is a benefit for users and we really should let them know that there is something.... and well... changed the manual on the Q.T.

Reply:
Aah, that's right. I misremembered.